ROADMAP.md

```diff
@@ -57,16 +57,6 @@ Currently we aim to align milestones with Data Study Groups at the Turing, so th
 
 The milestones can be seen [on GitHub](https://github.com/alan-turing-institute/data-safe-haven/milestones)
 
-## Long Term
-
-Long-term goals are developments which we have committed to but have not planned an expected completion date.
-This may be because they are lower priority than short-term goals or because the time required needs to be scoped.
-
-## Desired Features
-
-These are features we would like to highlight but which we have actively decided not to incorporate into the long- or short-term roadmap.
-This may be because, for example, we have decided we have insufficient resource, are lacking the right expertise or it is low priority.
-
 ### Develop Outside, Run Inside
```

Comment:
The section above this says that short-term goals will be in milestones. I think either this text should be in a milestone or the text above should be amended.

Reply:
This is in the milestone. I've removed it :+1:.